.github/workflows/build.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       - name: Run tests
         run: ctest --output-on-failure --parallel $(nproc)
         working-directory: ${{ github.workspace }}/build/linux/riscv64
-  cmake-linux-riscv64-rvv:
+  cmake-linux-riscv64-vector:
```

Comment:
I wonder why the suffix need changing. The summary didn't mention this as well.

Reply:
Because riscv64 already implies rv, the term rvv has been changed to a vector to make it more straightforward. The description has been updated to reflect these changes. Many thanks.

Reply:
The suffix has been changed back.